README.md

```diff
@@ -177,6 +177,31 @@ Call a gRPC endpoint using a JSON input. What this does behind the scenes:
 
 *See [docs/grpc.md](docs/grpc.md) for full instructions.*
 
+### SSL/TLS Connections
```

Comment:
Can you put this in `docs/grpc.md` instead?

Reply:
Absolutely, once we agree on final flags I'll move it there.